cmd/preview/deploy.go

```diff
@@ -57,6 +57,10 @@ func Deploy(ctx context.Context) *cobra.Command {
 				return okErrors.ErrNotLogged
 			}
 
+			if err := utils.SetOktetoUsernameEnv(); err != nil {
```

Comment:
I would do it as part of the global init function:
https://github.com/okteto/okteto/blob/master/main.go#L45